amaranth-chrome-ext/src/CalorieLabeller.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * An object which labels a dish name as low, average, or high-calorie.
+ */
+class CalorieLabeller {
+  /**
+   * Creates a CalorieLabeller object.
+   * @constructor
```

Comment:
Same here with the doc tag.

Reply:
Looking into it, this tag isn't required for JS class constructors. Will remove

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/a95b2263dbb2ca0b10330077196a4696d6f03db2